data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,244 @@ mission "Remnant: Visit to Paradise 1"
+			`	Down on the ground, you see Ruach and her team exit an airlock on the side of the Shattered Light and jump down to the ground. They gather around, giving hugs. "<first>!" exclaims Ruach. "We have recovered a lost piece of Remnant history. Thank you for helping. I do not know what this will mean for us in the long term, but there will be a lot of careful examination for the next few weeks."`
+			choice
+				`	"I have things to do, so just let me know if I can help."`
+					goto shatteredhelp
+				`	"So, what happened?"`
+			`	The team of Remnant are almost too excited for you to understand their gestures, but one calms down enough that their signing is comprehensible. "To be honest, we were expecting this to be the first of several trips doing detailed scans and trying out different sorts of synchronization fields. But when we saw that it was heading to the planet..." She trails off, and another one continues. "We already had the sensors running as we approached, so we jumped and triggered the fields as soon as we got a reasonable match."`
+			`	He frowns a bit. "It worked, although not perfectly. There are chunks missing, some of which seriously impair its functionality. But it is back in the primary weft, so now we have time to work on it and figure out what happened to it.`
+			choice
+				`	"I think I get the picture. Let me know if I can help."`
+					goto shatteredhelp
+				`	"What is this 'Primary Weft' you have mentioned?"`
+			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the secondary weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the primary weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort. Actually, it has been proposed that we call the Primary Weft the Warp. It is an old weaving term that serves as a reasonable analogy to how the warp and weft interact, and might better reflect its status instead of being labeled 'Primary.'`
```

Comment:
So we're introducing the terms Warp and Weft that were explained in the lore dump as being ideas from the Drak, but these don't seem to be the same definitions that were used there. To the Drak, the Warp and Weft were two different dimensions/planes of existence. But here, we have a Remnant saying that the "Primary Weft" and "Warp" are the same thing.

Perhaps the Remnant have these ideas of existence, but if they do I don't think they should be using the same terms that we know the Drak to have used (or at least whatever they would have said in the Drak language as it translates to English).

Reply:
The Remnant have different ideas about how things work, and I kind of like the idea of the lore doc version being more right than the Remnant one, but I deliberately want the cross-contamination of words & terms here.